### ISSUE TITLE
Question: what is "@mdx-js/tag" for in the next.js example?

### ISSUE THREAD

**Issue by trevordmiller** [mdx-js/mdx#129]
The [next.js example](https://github.com/mdx-js/mdx/tree/master/examples/next) shows a `"@mdx-js/tag"` dependency, but I don't see where this is being used or what it is for? It seems there isn't any documentation for this package: https://www.npmjs.com/package/@mdx-js/tag.

**Comment by timneutkens** [comment]
It’s included in the loader now, so it can be removed.

**Comment by revolunet** [comment]
looks like its still useful to provide custom components when parsing mdx, isn't it ?

**Comment by silvenon** [comment]
Yeah, it's where `MDXProvider` comes from, which is a way to specify `components` of all its MDX children.

**Comment by qburst-aparna** [comment]
The `MDXProvider ` components works only on SSR and it doesn't take the components set there for client side

**Comment by silvenon** [comment]
Really? Does it not work in development? Could you explain exactly what you mean and how to reproduce this?
